tests/signUpValidation.spec.js

```diff
@@ -0,0 +1,211 @@
+import {test, expect} from "@playwright/test";
+
+test.describe("Sign Up Form Validation", ()=>{
+    test.beforeEach(async ({page})=>{
+        await page.goto("https://qauto.forstudy.space/")
+        await page.click('text="Sign up"')
+    })
+
+    test.describe("Name Validation", ()=> {
+        test("Name is required", async ({page}) => {
+
+            const popup = page.locator('app-signup-modal')
+            const nameInput = popup.locator('#signupName')
+            const nameRequiredErrorMessage = popup.locator('.invalid-feedback')
```

Comment:
use more precise selectors

for example

`const nameRequiredErrorMessage = nameInput.locator("+ .invalid-feedback")`

Reply:
it is related to all tests